recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - setuptools
+    - python=3
+    - numpy=1.15
+    - scipy=1.1
+    - pandas=0.24
+    - astropy=3
+    - numexpr
+    - Cython=0.29
+    - networkx
+    - pyyaml
+    - jsonschema
+    - pyne=0.5
+    - pytables
+    - h5py
+    - jupyter
+    - notebook
+    - matplotlib
+    - graphviz
+    - pygraphviz
+    - sphinx
+    - nbconvert
+    - numpydoc
+    - docutils
+    - nbformat
+    - nbsphinx
+    - sphinx_bootstrap_theme
+    - sphinxcontrib-bibtex
+    - sphinx_rtd_theme
+    - git-lfs
+    - pytest
+    - pytest-html
+    - requests
+    - coverage
+    - docopt
+    - pytest-cov
+    - codecov
+    - backports
+
+  run:
```

Comment:
It looks like you probably have a lot of dependencies that the package doesn't actually need

Reply:
It's true. But for now I'm in charge only of the packaging (I'm new in the project).

Reply:
I mean things like `nbsphinx` are probably needed for building the documentation in your CI but not needed for users of the package.

My first guess would be you need the highlighted packages here (with the versions changed to `>=` rather than `=`): https://github.com/tardis-sn/tardis/blob/master/tardis_env3.yml#L8-L25